_component_design/loading-indicators.md

```diff
@@ -101,6 +101,10 @@ A variety of spinner icon sizes are available.
 
 {% endcolumn %}
 
+</div>
+
+<div class="hxRow" markdown="1">
```

Comment:
Any reason for adding these here? Typically, there's a left column and a right column within  one row div. This breaks them up so they don't line up properly.